donor-portal/src/pages/home/styles.css

```diff
@@ -112,3 +101,20 @@
     justify-content: center;
     align-items: center;
 }
+
+@media screen and (max-width: 900px) {
+    .home-description, .home-description-text,.current-donors, .logo-container{
+        flex: 1;
+        display: flex !important;
+        flex-direction: column !important;
+        align-items: center !important;
+        text-align: justify;
+        padding:1rem 1rem;
```

Comment:
I think we can just change the grid column layout to have a single column.

Also try to avoid `!important`

Reply:
Or may be to have two columns